docs/api/console.md

```diff
@@ -0,0 +1,123 @@
+# 🎮 console
+
+An implementation of the standard [`console`](https://developer.mozilla.org/en-US/docs/Web/API/console) API.
+
+## Quick start
+
+The console API provides access to the `jztz` debug log. 
+General purpose mesages can be created using [`console.log()`](#log)
+```typescript
+console.log("Hello from JS 👋");
+```
+For messages with a more specific purpose the following functions are provided:
+[`console.info`](#info), [`console.warn`](#warn), [`console.error`](#error), [`console.debug`](#debug).
+
+These functions can be combined to make logs more informative.
+```typescript
+function myAwesomeFunction(arg: unknown) {
+    console.debug(`Entered myAwesomeFunction with argument ${arg}`)
+    if (typeof arg !== 'string') {
+    console.warn("Converting non string argument to string");
+    arg = arg.toString();
+    }
+    try {
+        let result = functionThatMayThrow(arg);
+        console.info(`Result: ${result}`);
+    } catch (error) {
+        console.error(`Call failed with message ${error}`);
+    }
+}
+```
+## Logging objects 
+`console` functions are able to output a string representation for standard javascript objects.
+All logging functions can be called with multiple arguments, the message will consist of the string representations of each of the arguments
+```
+const dave = { name: "Dave", age: 42 };
+console.log("Hello", dave);
+```
+
+In practice using multiple arguments is unwieldy and difficult to read.
```

Comment:
This should be a subsection.

```suggestion
## Template Literals

In practice using multiple arguments is unwieldy and difficult to read.
```

Reply:
Additionally I'm not too convinced we need this section. Template literals are part of JavaScript, not jstz.